sparkle/src/components/Tree.tsx

```diff
@@ -49,13 +49,15 @@ interface TreeItemProps {
   label?: string;
   type?: "node" | "item" | "leaf";
   variant?: "file" | "folder" | "database" | "channel";
-  visual?: React.ReactNode;
+  size?: "sm" | "md";
+  visual?: ComponentType<{ className?: string }>;
```

Comment:
why this change ?

Reply:
@fontanierh Yeah I don't know either, this is actually what breaks in https://github.com/dust-tt/dust/pull/6607.

Reply:
Dunno what was the rational behind but it could be to render all visuals as `Icon` and control the size?